test/test_utils_config_module.py

```diff
@@ -395,6 +395,14 @@ def test_reference_is_default(self):
         t["a"] = "b"
         self.assertFalse(config._is_default("e_dict"))
 
+    def testInvalidConfig(self):
```

Comment:
FYI - tests are not passing, let me know if you need help reproing the failure locally on your machine.

Reply:
Yeah I figured out how to repro it earlier, I was testing on _Config instead of _ConfigModule and the logic was also not right.